app/Jobs/SendEmptyWikibaseNotificationsJob.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Notifications\EmptyWikibaseNotification;
+use App\Wiki;
+use Carbon\CarbonImmutable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Support\Facades\Log;
+
+class SendEmptyWikibaseNotificationsJob extends Job implements ShouldBeUnique
+{
+    public function handle (): void
+    {
+        $allWikis = Wiki::all();
```

Comment:
I think this would still need to load the `wikiManagers` and `wikibaseNotificationSentRecord` relations.

Reply:
from Dat and I chatting: maybe something like https://laravel.com/docs/8.x/eloquent-relationships#eager-loading would be something to look at

Reply:
I didn't even know lazy loading existed.